armbuild.Dockerfile

```diff
@@ -0,0 +1,53 @@
+FROM arm32v7/node:12-alpine as dependency-builder
```

Comment:
The armbuild Dockerfile wasn't making a dungeon-revealer image to be run on arm machines. It was just an emulated arm environment to compile the binary. We should build production images for other architectures though. I've been reading about [buildx](https://www.docker.com/blog/multi-platform-docker-builds/) which can do this automatically.

Reply:
Oh okay, so should I revert this file for now? The only contents required would be `FROM arm32v7/node:12-alpine`? Could we even reference `arm32v7/node:12-alpine` in the command without having this dockerfile inside the repository?

Reply:
Yes, we can. We'd have to mount the source directory as a volume. The arm build would have to be in its own job to prevent artifacts from other architectures causing problems (We already have it set up this way).

In build-binaries.yml, the arm build job should look like this:
```
  job_build_arm:
    name: ARM Linux Binaries
    runs-on: ubuntu-latest
    steps:
      - name: Checkout code
        uses: actions/checkout@v2

      - name: Build arm
        run: |
            docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
            docker run --rm -v $(pwd):/usr/src/app/ arm32v7/node:12 /bin/sh -c 'cd /usr/src/app/; yarn install; yarn compilearm'
            cp bin/arm/dungeon-revealer bin/dungeon-revealer-linux-armv7
            rm -rf bin/arm

      - name: Upload artifacts
        uses: actions/upload-artifact@v1
        with:
          name: binaries
          path: bin/
```

I'm using the debian based image of node because the alpine images don't have all the build tools installed by default. Specifically, sqlite3 complains that there is no python.

Reply:
```
 sudo docker run --rm --privileged multiarch/qemu-user-static:register
          wget https://github.com/multiarch/qemu-user-static/releases/download/v4.1.1-1/qemu-arm-static.tar.gz -O /tmp/qemu-arm-static.tar.gz
          tar zxvf /tmp/qemu-arm-static.tar.gz -C /tmp
```

What about this stuff?

Reply:
That's not needed anymore. The build environments have QEMU set up properly. I'm working on adding multiarch docker builds to this and ironing out the travis bugs. See [maxb2:docker-multi-arch](https://github.com/maxb2/dungeon-revealer/tree/docker-multi-arch).

Reply:
ok nice!